nextflow_schema.json

```diff
@@ -453,6 +453,11 @@
                     "default": 1000,
                     "description": "Discard any read which is shorter than this value."
                 },
+                "longreads_min_quality": {
```

Comment:
Is this nanoq only? If so we should ideally specify this both in the parameter name and the description

Reply:
I added this in filtlong as well, if the user specifies ``--longreads_min_quality`` and ``--longread_filtering-tool filtlong``, filtlong will now also filter by mean quality, if  ``--longreads_min_quality`` is not added, filtlong runs as before

Reply:
OK all good :+1: